modules/client-configuration/pages/clients-amazon.adoc

```diff
@@ -84,11 +84,21 @@ The channels you need for this procedure are:
 | amazonlinux2-core
 | amazonlinux2-uyuni-client
 
+| {amazon} 2023
+| amazonlinux2023
```

Comment:
Strictly speaking, this is not called `core` anymore, unlike on Amazon Linux 2, but I guess we can keep the title for the table as it is?

Reply:
ok, thanks!

Reply:
Done.